<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FigWidget</class>
 <widget class="QWidget" name="FigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1103</width>
    <height>713</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>540</y>
     <width>791</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>路线显示</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="PathList">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>20</horstretch>
        <verstretch>92</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>769</width>
        <height>117</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>769</width>
        <height>117</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="midLineWidth">
       <number>1</number>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustIgnored</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="PathListContent">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>767</width>
         <height>115</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="RightFrame_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>281</width>
     <height>698</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>100</verstretch>
    </sizepolicy>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="TravelerLayout1">
          <item>
           <widget class="QPushButton" name="addTravelerButton">
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>59</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>添加旅客</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="TravelerLayout">
            <item>
             <widget class="QLabel" name="TravelerChoseLabel">
              <property name="text">
               <string>旅客：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="TravelerComboBox"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="ComboLayout2">
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>12</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="StartCityLayout">
            <property name="spacing">
             <number>7</number>
            </property>
            <property name="sizeConstraint">
             <enum>QLayout::SetFixedSize</enum>
            </property>
            <item>
             <widget class="QLabel" name="StartLabel">
              <property name="text">
               <string>始发地：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="StartComboBox">
              <item>
               <property name="text">
                <string>北京</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>哈尔滨</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>长春</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>青岛</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>上海</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>武汉</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>广州</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>西安</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>西宁</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>乌鲁木齐</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>拉萨</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>成都</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>12</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="DestinationCityLayout">
            <item>
             <widget class="QLabel" name="DestinationLabel">
              <property name="text">
               <string>目的地：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="DestinationComboBox">
              <item>
               <property name="text">
                <string>北京</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>哈尔滨</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>长春</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>青岛</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>上海</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>武汉</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>广州</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>西安</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>西宁</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>乌鲁木齐</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>拉萨</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>成都</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>12</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QGroupBox" name="StrategyChooseBox">
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>100</height>
             </size>
            </property>
            <property name="title">
             <string>策略选择</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QRadioButton" name="StrategyBtn1">
               <property name="text">
                <string>最小风险</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="StrategyBtn2">
               <property name="text">
                <string>限时最小风险</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="TimeEditLayout4">
        <item>
         <layout class="QHBoxLayout" name="CurrentTimeLayout">
          <item>
           <widget class="QLabel" name="TimeLabel">
            <property name="text">
             <string>起始时间：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateTimeEdit" name="StartDateTimeEdit">
            <property name="dateTime">
             <datetime>
              <hour>0</hour>
              <minute>0</minute>
              <second>0</second>
              <year>2020</year>
              <month>1</month>
              <day>1</day>
             </datetime>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="DeadlineLayout">
          <item>
           <widget class="QLabel" name="DeadlineLabel">
            <property name="text">
             <string>截止时间：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateTimeEdit" name="DeadlineDateTimeEdit">
            <property name="dateTime">
             <datetime>
              <hour>0</hour>
              <minute>0</minute>
              <second>0</second>
              <year>2020</year>
              <month>1</month>
              <day>1</day>
             </datetime>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="StartButtonLayout5">
        <item>
         <widget class="QPushButton" name="StartButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>250</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>59</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>开始规划路线</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_6">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Preferred</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="FareTimeDurLayout7">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>行程进度：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="progressBar">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="TotalFareLayout">
        <item>
         <widget class="QLabel" name="FareLabel">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>风险评估：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="FareEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="TotalTimeLayout">
        <item>
         <widget class="QLabel" name="TotalTimeLabel">
          <property name="text">
           <string>预计时间：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="TotalTimeEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="DurationLayout">
        <item>
         <widget class="QLabel" name="DurationLabel">
          <property name="text">
           <string>已用时间：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="DurationText"/>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="CurTimeLabel">
          <property name="text">
           <string>当前时间：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="CurTimeText">
          <property name="contextMenuPolicy">
           <enum>Qt::NoContextMenu</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>15</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>旅客移动倍速</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ChangeSpeed" name="spinAndSlider" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>100</horstretch>
            <verstretch>20</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_5">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Preferred</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>30</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="figBackBtn">
        <property name="text">
         <string>返回</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exitBtn">
        <property name="text">
         <string>退出</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="Gif" name="Map" native="true">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>-1</y>
     <width>791</width>
     <height>541</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Gif</class>
   <extends>QWidget</extends>
   <header location="global">gif.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ChangeSpeed</class>
   <extends>QWidget</extends>
   <header>changespeed.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
